File: python-js/retention-assignments/algo-sum-pairs/js/sumPairs.js
sumPairs = function(numberList, goal) {
    let ret = [];
    for (let i = 0; i < numberList.length; i++) {
        for (let j = i+1; j < numberList.length; j++) {
            if (numberList[i] + numberList[j] == goal) {
                ret.push([numberList[i], numberList[j]]);
            }
        }
    }
    return ret;
};

module.exports = sumPairs;

File: python-js/intro-to-code/retention-assignments/algo-sum-pairs/python/sum_pairs.py
def sum_pairs(number_list, goal):
    ret = []
    for i in range(len(number_list)):
        for j in range(i+1, len(number_list)):
            if (number_list[i] + number_list[j]) == goal:
                ret.append([number_list[i], number_list[j]])
    return ret